### PR TITLE
Update root level README

### PR DESCRIPTION
**Pull request by aayushmahapatra**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,64 +22,40 @@
   </a>
 </p>
 
+---
+
+The Spheron developer toolkits include two packages that can help you build decentralized applications (dapps) on the web3 network:
```
Comment (shomix) [inline README.md]: I would just remove the `two` word, so
`The Spheron developer toolkits includes packages...`.
Just so we do not need to update it the next time we add a new package :D

**Review by shomix — APPROVED** [review]
LGTM! One suggestion

File: README.md (hunk 1 above)
Comment (aayushmahapatra) [reply README.md]: Sure 😂

**Review by shomix — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -22,64 +22,40 @@
   </a>
 </p>
 
+---
+
+Spheron's developer toolkit includes packages that makes it easy to build decentralized applications (dapps):
```
Comment (shomix) [inline README.md:27]: Wording seems a bit off, does this make more sense maybe?
```suggestion
Spheron's developer toolkit includes packages that make it easier to build decentralized applications (dapps):
```

**Review by dusanstanis — DISMISSED** [review]

**Review by shomix — APPROVED** [review]

**Review by dusanstanis — APPROVED** [review]

**Pull request merged**